python/src/DickeyFullerTest_doc.i.in

```diff
@@ -174,3 +174,23 @@ Returns
 testResult : :class:`~openturns.TestResult`
     Results container of the tests. The strategy if the one described above.
 "
+
+// ---------------------------------------------------------------------
+%feature("docstring") OT::DickeyFullerTest::setVerbose
```

Comment:
you can drop these, I'm deprecating DickeyFullerTest::setVerbose & getVerbose

Reply:
ok...